spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/TestSparkDistributionAndOrderingUtil.java

```diff
@@ -1567,6 +1567,370 @@ public void testRangePositionDeltaUpdatePartitionedTable() {
         table, UPDATE, expectedDistribution, SPEC_ID_PARTITION_FILE_POSITION_ORDERING);
   }
 
+  // ==================================================================================
+  // Distribution and ordering for merge-on-read MERGE operations with position deletes
+  // ==================================================================================
+  //
+  // UNPARTITIONED UNORDERED
+  // -------------------------------------------------------------------------
+  // merge mode is NOT SET -> rely on write distribution and ordering as a basis
```

Comment:
I think by this you mean that we will check the write distribution mode and then use one of the following cases, right? So we can expect this to be `unspecified distribution + LOCALLY ORDER BY _spec_id, _partition, _file, _pos` because all 3 modes (whether merge or write mode) have that behavior?

Reply:
Correct, I just did not want to duplicate all possible values of the write property. If merge mode is not set, use the write mode and adapt it to cover deletes.